docs/docroot/index.html

```diff
@@ -1 +1,11 @@
-<html><body><h1>It works!</h1></body></html>
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+		<title>Welcome to Apache Server 2.4</title>
```

Comment:
If you add an opening `<head>` tag, please also add a closing one.

Reply:
DONE